### PR TITLE
Update of README

### PR DESCRIPTION
**Pull request by IgorBia**
Changed:
    - Deleted unnecessary sentence at Tomcat part and repeated words in Content part,
    - Fixed misspelling of branch names (releases/<version> -> release/<version>),
    - Fixed link to maven repository (previous link returns a 400 Error),
    - Improved visibility of text in `xyz` apostrophes by bolding.
Added: 
    - Table of contents, 
    - Content point, 
    - Xml formatting, 
    - Connecting with ACA point, 
    - Helpful links point.

### CODE REVIEW

**Review by damianujma — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -64,34 +84,51 @@ The artifacts can be obtained by:
...
-### Contributing guide
-Please use [this guide](CONTRIBUTING.md) to make a contribution to the project.
+## Contributing guide
+
+Please use [**this guide**](CONTRIBUTING.md) to make a contribution to the project.
+
+## Helpful links
+
+- [Alfresco Content Services Documentation ](https://docs.alfresco.com/content-services/latest/)
+- [Alfresco Content Application Repository](https://github.com/Alfresco/alfresco-content-app)
```
Comment (dsibilio) [inline README.md:133]:
```suggestion
```

There's many optional components that can be integrated with ACS. I wouldn't go out of our way to suggest on how to integrate with ACA here, so I'd remove this link + the related section.

**Review by dsibilio — COMMENTED** [review]
First and foremost, thank you very much for your contribution.

Nice clean-up, I have left just a few comments I'd like to see addressed.

File: README.md (hunk 1 above)
Comment (IgorBia) [reply README.md]: Hi, I understand that ACA is optional and not necessary for every use - I reedited what you've suggested and I think everything is OK now.

**Review by dsibilio — APPROVED** [review]

**Comment by dsibilio** [comment]
Thanks once again for your contribution. Looks good now 🙏

**Pull request merged**